### DISCUSSION TITLE
好奇为啥从 Jest 迁移到 Vitest，又 revert 回 Jest 了？

### DISCUSSION THREAD

**Discussion by mortalYoung** [ant-design/ant-design#47233] [General]
# 从 Jest 迁移到 Vitest
我理解最大的原因应该是 Vitest 很快吧？除此之外，还有其他原因么？

# Rollback
revert 的原因是啥？是用 Vitest 遇到无法解决的问题？还是说不顺手？

希望不吝赐教。

**Comment by li-jia-nan** [comment]
case 太多的时候，Vitest 会有内存泄漏的现象

**Comment by zombieJ** [comment]
因为部分社区同学的机器比较旧，vitest 会挂掉。。。

**Comment by RedJue** [comment]
总体而言还是当时迁移的时候 vitest 不够稳定，大部分机子都没法跑成功，所以回滚了。

**Comment by fengmk2** [comment]
还有机会迁移到 vitest 么，哈哈，我想来再尝试一次

**Reply by afc163** [reply]
来试试

**Reply by Wxh16144** [reply]
蹲一手，虽然当前 jest 在 #54576 提升到了两分钟内，不知道 vitest 能不能更上一层楼。
https://x.com/Wxh16144/status/1953138080803684661

**Reply by fengmk2** [reply]
> 不知道 vitest 能不能更上一层楼

周末先尝试一下，不够快就放弃。

**Reply by fengmk2** [reply]
CC 跑了 2 天，感觉改动量太大了，几乎无法迁移 https://github.com/fengmk2/ant-design/pull/1/changes#diff-32c327bf0beb58be3a93687a221c8de7efa2309ce1a8adaec45e7dc88b1dae4c

**Reply by afc163** [reply]
看来我和 CC 一样强
